Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -384,6 +384,12 @@ extension TelemetryWrapper {
         case onboardingSecondaryButton = "onboarding-card-secondary-button"
         case onboardingSelectWallpaper = "onboarding-select-wallpaper"
         case onboarding = "onboarding"
+        // MARK: New Upgrade screen
+        case upgradeClose = "upgrade-close"
```

Comment:
would it be more clear if these were called 'upgradeOnboarding...'

Reply:
I will update the cases